TerrorConsole/Assets/_Main/Scripts/Input/InputManager.cs

```diff
@@ -44,5 +46,17 @@ private void SetMovementInput()
             
             IsMoving = _movementDirection is not { x: 0, y: 0 };
         }
+
+        private void SetActionButtons()
+        {
+            if (Input.GetKeyDown(KeyCode.M))
+            {
+                ActionButton1 = true;
+            }
+            else
+            {
+                ActionButton1 = false;
+            }
```

Comment:
Since this will be an event you dont need the ´else´ and the content of the ´if´ should look like ´ OnActivateCloseObject?.Invoke()´